lock/src/main/java/com/auth0/android/lock/LockActivity.java

```diff
@@ -91,17 +98,43 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             return;
         }
 
+        getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE);
         Bus lockBus = new Bus();
         lockBus.register(this);
         handler = new Handler(getMainLooper());
 
         setContentView(R.layout.com_auth0_lock_activity_lock);
+        ViewGroup rootContainer = (ViewGroup) findViewById(R.id.com_auth0_lock_container);
+        headerView = (HeaderView) findViewById(R.id.com_auth0_lock_header);
         resultMessage = (TextView) findViewById(R.id.com_auth0_lock_result_message);
         RelativeLayout rootView = (RelativeLayout) findViewById(R.id.com_auth0_lock_content);
         panelHolder = new ClassicPanelHolder(this, lockBus);
         rootView.addView(panelHolder, ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT);
 
         lockBus.post(new FetchApplicationEvent());
+        setupKeyboardListener(rootContainer);
+    }
+
+    private void setupKeyboardListener(final ViewGroup container) {
+        container.getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
```

Comment:
Yes! But there's no method to remove it prior to API 16. Anyway, making it null before `onDestroy` should prevent possible memory leaks in both cases.